commons-test/src/main/java/org/infinispan/commons/test/EnvironmentCheck.java

```diff
@@ -0,0 +1,20 @@
+package org.infinispan.commons.test;
+
+/**
+ * Verifies that the tests are being run with a JVM version which matches the supplied regex
+ *
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.0
+ **/
+public class EnvironmentCheck {
+
+   public static final void checkJVMVersion() {
+      String jvmVersionRegex = System.getProperty("test.jvm.version.regex");
```

Comment:
The system properties that we access from Java (as opposed to Maven-only) usually start with `infinispan.test`

Reply:
Sure

Reply:
You can now do:
```
mvn verify -Pjava8-test -Dinfinispan.test.jvm.version.regex="^2.*"
```